src/checkpoint/checkpoint.h

```diff
@@ -197,15 +197,16 @@ extern int __wt_checkpoint_server_destroy(WT_SESSION_IMPL *session)
   WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
 extern int __wt_checkpoint_sync(WT_SESSION_IMPL *session, const char *cfg[])
   WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
+extern void __wt_checkpoint_apply_or_skip_handle_update(WT_SESSION_IMPL *session, uint64_t time_us);
 extern void __wt_checkpoint_free(WT_SESSION_IMPL *session, WT_CKPT *ckpt);
 extern void __wt_checkpoint_progress(WT_SESSION_IMPL *session, bool closing);
-extern void __wt_checkpoint_reset_handle_stats(WT_SESSION_IMPL *session, WT_CKPT_CONNECTION *ckpt);
-extern void __wt_checkpoint_set_handle_stats(
-  WT_SESSION_IMPL *session, WT_CKPT_CONNECTION *ckpt, uint64_t gathering_handles_time_us);
+extern void __wt_checkpoint_reset_handle_stats(WT_SESSION_IMPL *session);
+extern void __wt_checkpoint_reset_timer_stats(WT_SESSION_IMPL *session);
```

Comment:
Again for consistency with the function names I'd suggest this one to be `__wt_checkpoint_stats_reset_timer` or `__wt_checkpoint_stats_timer_reset`. Just keeping the prefix `__wt_checkpoint_stats` consistent.